packages/flutter_sprites/lib/src/node.dart

```diff
@@ -769,10 +769,10 @@ class Node {
   ///       }
   ///
   ///       bool handleEvent(SpriteBoxEvent event) {
-  ///         if (event.type == 'pointerdown) {
+  ///         if (event.type == PointerDownEvent) {
```

Comment:
I think "event.type == PointerDownEvent" should be "event is PointerDownEvent" here?